crawler/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta encoding='utf-8'/>
+    <title>Headless detector</title>
+  </head>
+
+  <body>
+    <table>
+        <tr>
+            <th>Test</th>
+            <th>Result</th>
+        </tr>
+        <tbody id="results"></tbody>
+    </table>
+    <template id="result-row">
+        <tr>
+            <td class="key"></td>
+            <td class="value"></td>
+        </tr>
+    </template>
+    <script>
+        (async () => {
+            const ul = document.getElementById('results');
+            const template = document.getElementById('result-row');
+            
+            function displayResult(test, result, detectValue) {
+                const listEntry = template.content.cloneNode(true);
+                listEntry.querySelector('.key').textContent = test;
+                listEntry.querySelector('.value').textContent = result ? 'fail' : 'pass';
+                ul.appendChild(listEntry)
+                if (result) {
+                    fetch(`./fail.json?test=${test}&${detectValue}`);
```

Comment:
I like this :) I'd add a note that this is used for easy access to test results from the crawler's output.